### PR TITLE
[PIR] open test in test_ifelse for PIR

### PR DESCRIPTION
**Pull request by chen2016013**
### PR types

Others

### PR changes

Others

### Description

Pcard-67164

### CODE REVIEW

File: test/dygraph_to_static/ifelse_simple_func.py [hunk 1]
```diff
@@ -60,9 +60,12 @@ def dyfunc_with_if_else2(x, col=100):
         # TODO: Don't support return non-Tensor in Tensor-dependent `if` statement currently.
         #  `x` is Tensor, `col` is not Tensor, and `col` is the return value of `true_fn` after transformed.
         # col = -1
-        col = paddle.tensor.fill_constant(shape=[1], value=-1, dtype="int64")
+        col = paddle.tensor.fill_constant(shape=[], value=-1, dtype="int64")
```
Comment (zhangbo9674) [inline test/dygraph_to_static/ifelse_simple_func.py:63]:
```suggestion
        col = paddle.tensor.fill_constant(shape=[1], value=-1, dtype="int64")
```

File: test/dygraph_to_static/ifelse_simple_func.py [hunk 2]
```diff
@@ -60,9 +60,12 @@ def dyfunc_with_if_else2(x, col=100):
...
+    else:
+        col = paddle.tensor.fill_constant(shape=[], value=1, dtype="int64")
```
Comment (zhangbo9674) [inline test/dygraph_to_static/ifelse_simple_func.py:65]:
```suggestion
        col = paddle.tensor.fill_constant(shape=[1], value=1, dtype="int64")
```

File: test/dygraph_to_static/ifelse_simple_func.py [hunk 3]
```diff
@@ -223,12 +250,14 @@ def nested_if_else_3(x):
...
             # `z` is created in above code block.
             z = y + 1
+            out = x + 1
         else:
             res = paddle.tensor.fill_constant(
                 value=3, shape=x.shape, dtype="int32"
             )
             # `out` is a new var.
             out = x + 1
+            z = y + 1
```
Comment (zhangbo9674) [inline test/dygraph_to_static/ifelse_simple_func.py]: 建议两个分支有一些区别？

Comment (chen2016013) [reply test/dygraph_to_static/ifelse_simple_func.py]: done

**Review by zhangbo9674 — APPROVED** [review]
LGTM

**Pull request merged**